docs/building_fledge/building_fledge.rst

```diff
@@ -561,48 +561,50 @@ In order to use the latest version for Fledge, add the following lines at the en
   export LD_LIBRARY_PATH=/usr/local/lib64
 
 
-Installing PostgreSQL 9.6
+Installing PostgreSQL 13 
 -------------------------
 
-CentOS provides PostgreSQL 9.2. Fledge has been tested with PostgreSQL 9.5, 9.6 and 10.X.
+CentOS provides PostgreSQL 9.2. Fledge has been tested with PostgreSQL 13, 9.5, 9.6 and 10.X.
 Following https://www.postgresql.org/download/ instructions, the commands to install the new version of PostgreSQL are:
 
 .. code-block:: console
 
   sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
-  sudo yum install -y postgresql96-server
-  sudo yum install -y postgresql96-devel
-  sudo yum install -y rh-postgresql96
-  sudo yum install -y rh-postgresql96-postgresql-devel
-  sudo /usr/pgsql-9.6/bin/postgresql96-setup initdb
-  sudo systemctl enable postgresql-9.6
-  sudo systemctl start postgresql-9.6
+  sudo yum install -y postgresql13-server
+  sudo yum install -y postgresql13-devel
+  sudo yum install -y rh-postgresql13
+  sudo yum install -y rh-postgresql13-postgresql-devel
+  sudo /usr/pgsql-13/bin/postgresql13-setup initdb
```

Comment:
`sudo /usr/pgsql-13/bin/postgresql13-setup initdb` to `sudo /usr/pgsql-13/bin/postgresql-13-setup initdb`

Reply:
fixed